tests/functional/settings_secondary_emails.js

```diff
@@ -144,6 +152,86 @@ define([
         .then(openPage(SIGNUP_URL, '#fxa-signup-header'))
         .then(fillOutSignUp(email, PASSWORD))
         .then(testElementExists('#fxa-settings-content'));
+    },
+
+    'unblock code is sent to secondary emails': function () {
+      email = TestHelpers.createEmail('blocked{id}');
+
+      return this.remote
+        .then(createUser(email, PASSWORD, { preVerified: true }))
+        .then(function (result) {
+          return client.recoveryEmailCreate(result.sessionToken, secondaryEmail);
+        })
+        .then(fillOutSignIn(email, PASSWORD))
+        .then(getUnblockInfo(email, 0))
+        .then(testElementTextInclude('.verification-email-message', email))
+        .then(getUnblockInfo(email, 0))
+        .then(function (unblockInfo) {
+          return this.parent
+            .then(type('#unblock_code', '   ' + unblockInfo.unblockCode));
+        })
+        .then(click('button[type=submit]'))
+
+        .then(testElementExists('#fxa-settings-header'))
+        .then(openVerificationLinkInSameTab(secondaryEmail, 0))
+        .then(click('#emails .settings-unit-stub button'))
+        .then(testElementExists('.verified'))
+        .then(click('#signout'))
+        .then(fillOutSignIn(email, PASSWORD))
+        .then(testElementTextInclude('.verification-email-message', email))
+        .then(getUnblockInfo(email, 0))
+        .then(function (unblockInfo) {
+          // original email gets the unblock code
+          assert.ok(unblockInfo.unblockCode);
+        })
+        .then(getUnblockInfo(secondaryEmail, 1))
+        .then(function (unblockInfo) {
+          return this.parent
```

Comment:
@vbudhram both unblock codes and confirmation do not arrive for me when I wrote these against the `latest` server?  Any ideas why?

For example, the account `sync0.36463992740027606@restmail.net` has a verified secondary at `signin0.8020127578638494@restmail.net`, default `password` (on `latest.dev`). 

However, manually testing that account and the emails are not arriving to http://restmail.net/mail/signin0.8020127578638494